### PR TITLE
fix bug #474

### PR DESCRIPTION
**Pull request by sophiesongge**
for scheduling portal not reporting correct message when a user is not authorized to fetch the job output

### CODE REVIEW

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 10 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [OutputController.java#L186](https://github.com/ow2-proactive/scheduling-portal/blob/61387a6e221056be9d5c20be16855254407bb855/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L186): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [OutputController.java#L216](https://github.com/ow2-proactive/scheduling-portal/blob/61387a6e221056be9d5c20be16855254407bb855/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L216): Merge this if statement with the enclosing one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)
1. ![MAJOR][MAJOR] [AbstractOutputDisplayView.java#L90](https://github.com/ow2-proactive/scheduling-portal/blob/61387a6e221056be9d5c20be16855254407bb855/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/AbstractOutputDisplayView.java#L90): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [OutputView.java#L96](https://github.com/ow2-proactive/scheduling-portal/blob/61387a6e221056be9d5c20be16855254407bb855/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L96): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [OutputView.java#L108](https://github.com/ow2-proactive/scheduling-portal/blob/61387a6e221056be9d5c20be16855254407bb855/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L108): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MINOR][MINOR] [OutputController.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/61387a6e221056be9d5c20be16855254407bb855/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)
1. ![MINOR][MINOR] [OutputController.java#L78](https://github.com/ow2-proactive/scheduling-portal/blob/61387a6e221056be9d5c20be16855254407bb855/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L78): Replace the type specification in this constructor call with the diamond operator ("<>"). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)
1. ![MINOR][MINOR] [OutputController.java#L358](https://github.com/ow2-proactive/scheduling-portal/blob/61387a6e221056be9d5c20be16855254407bb855/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L358): Replace the type specification in this constructor call with the diamond operator ("<>"). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)
1. ![MINOR][MINOR] [AbstractOutputDisplayView.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/61387a6e221056be9d5c20be16855254407bb855/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/AbstractOutputDisplayView.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)
1. ![MINOR][MINOR] [OutputView.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/61387a6e221056be9d5c20be16855254407bb855/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 10 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [OutputController.java#L180](https://github.com/ow2-proactive/scheduling-portal/blob/b803cbe32996cd39409b2cf1a222d3546a29372b/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L180): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [OutputController.java#L210](https://github.com/ow2-proactive/scheduling-portal/blob/b803cbe32996cd39409b2cf1a222d3546a29372b/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L210): Merge this if statement with the enclosing one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)
1. ![MAJOR][MAJOR] [AbstractOutputDisplayView.java#L90](https://github.com/ow2-proactive/scheduling-portal/blob/b803cbe32996cd39409b2cf1a222d3546a29372b/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/AbstractOutputDisplayView.java#L90): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [OutputView.java#L96](https://github.com/ow2-proactive/scheduling-portal/blob/b803cbe32996cd39409b2cf1a222d3546a29372b/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L96): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [OutputView.java#L108](https://github.com/ow2-proactive/scheduling-portal/blob/b803cbe32996cd39409b2cf1a222d3546a29372b/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L108): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MINOR][MINOR] [OutputController.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/b803cbe32996cd39409b2cf1a222d3546a29372b/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)
1. ![MINOR][MINOR] [OutputController.java#L78](https://github.com/ow2-proactive/scheduling-portal/blob/b803cbe32996cd39409b2cf1a222d3546a29372b/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L78): Replace the type specification in this constructor call with the diamond operator ("<>"). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)
1. ![MINOR][MINOR] [OutputController.java#L352](https://github.com/ow2-proactive/scheduling-portal/blob/b803cbe32996cd39409b2cf1a222d3546a29372b/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L352): Replace the type specification in this constructor call with the diamond operator ("<>"). [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2293)
1. ![MINOR][MINOR] [AbstractOutputDisplayView.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/b803cbe32996cd39409b2cf1a222d3546a29372b/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/AbstractOutputDisplayView.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)
1. ![MINOR][MINOR] [OutputView.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/b803cbe32996cd39409b2cf1a222d3546a29372b/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 18 issues
* ![MAJOR][MAJOR] 9 major
* ![MINOR][MINOR] 9 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [OutputController.java#L162](https://github.com/ow2-proactive/scheduling-portal/blob/a27c30c29509f015c2fe19d33fbde5321ca2b1b5/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L162): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [OutputController.java#L192](https://github.com/ow2-proactive/scheduling-portal/blob/a27c30c29509f015c2fe19d33fbde5321ca2b1b5/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L192): Merge this if statement with the enclosing one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L95](https://github.com/ow2-proactive/scheduling-portal/blob/a27c30c29509f015c2fe19d33fbde5321ca2b1b5/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L95): Remove this useless assignment to local variable "msg". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L97](https://github.com/ow2-proactive/scheduling-portal/blob/a27c30c29509f015c2fe19d33fbde5321ca2b1b5/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L97): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L145](https://github.com/ow2-proactive/scheduling-portal/blob/a27c30c29509f015c2fe19d33fbde5321ca2b1b5/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L145): Remove this useless assignment to local variable "msg". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L147](https://github.com/ow2-proactive/scheduling-portal/blob/a27c30c29509f015c2fe19d33fbde5321ca2b1b5/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L147): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [AbstractOutputDisplayView.java#L88](https://github.com/ow2-proactive/scheduling-portal/blob/a27c30c29509f015c2fe19d33fbde5321ca2b1b5/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/AbstractOutputDisplayView.java#L88): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [OutputView.java#L93](https://github.com/ow2-proactive/scheduling-portal/blob/a27c30c29509f015c2fe19d33fbde5321ca2b1b5/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L93): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [OutputView.java#L105](https://github.com/ow2-proactive/scheduling-portal/blob/a27c30c29509f015c2fe19d33fbde5321ca2b1b5/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L105): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MINOR][MINOR] [OutputController.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/a27c30c29509f015c2fe19d33fbde5321ca2b1b5/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

File: scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java [hunk 1]
```diff
@@ -73,6 +73,9 @@ public OutputView(OutputController controller) {
 
         this.refreshButtonLabel = "Finished Tasks Output";
         this.refreshButtonTooltip = "Request fetching the logs for finished tasks of the entire Job, or for a selected Task";
+
+        this.notAuthorized = "You are not authorized to retrieve this job's output";
```
Comment (Mykhailenko) [inline scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java]: This field does not look like a variable. Maybe you can remove this field at all and extract constant?

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 18 issues
* ![MAJOR][MAJOR] 9 major
* ![MINOR][MINOR] 9 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [OutputController.java#L180](https://github.com/ow2-proactive/scheduling-portal/blob/786fcc7bf160e4962e89607ea3892235237e28f2/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L180): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [OutputController.java#L210](https://github.com/ow2-proactive/scheduling-portal/blob/786fcc7bf160e4962e89607ea3892235237e28f2/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L210): Merge this if statement with the enclosing one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L95](https://github.com/ow2-proactive/scheduling-portal/blob/786fcc7bf160e4962e89607ea3892235237e28f2/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L95): Remove this useless assignment to local variable "msg". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L97](https://github.com/ow2-proactive/scheduling-portal/blob/786fcc7bf160e4962e89607ea3892235237e28f2/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L97): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L148](https://github.com/ow2-proactive/scheduling-portal/blob/786fcc7bf160e4962e89607ea3892235237e28f2/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L148): Remove this useless assignment to local variable "msg". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L150](https://github.com/ow2-proactive/scheduling-portal/blob/786fcc7bf160e4962e89607ea3892235237e28f2/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L150): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [AbstractOutputDisplayView.java#L90](https://github.com/ow2-proactive/scheduling-portal/blob/786fcc7bf160e4962e89607ea3892235237e28f2/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/AbstractOutputDisplayView.java#L90): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [OutputView.java#L96](https://github.com/ow2-proactive/scheduling-portal/blob/786fcc7bf160e4962e89607ea3892235237e28f2/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L96): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [OutputView.java#L108](https://github.com/ow2-proactive/scheduling-portal/blob/786fcc7bf160e4962e89607ea3892235237e28f2/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L108): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MINOR][MINOR] [OutputController.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/786fcc7bf160e4962e89607ea3892235237e28f2/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Comment by sonarqube-activeeon** [comment]
SonarQube analysis reported 18 issues
* ![MAJOR][MAJOR] 9 major
* ![MINOR][MINOR] 9 minor

#### Top 10 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [OutputController.java#L162](https://github.com/ow2-proactive/scheduling-portal/blob/962554aa96e9eee886cc1da34cf8ad30ae790cba/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L162): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [OutputController.java#L192](https://github.com/ow2-proactive/scheduling-portal/blob/962554aa96e9eee886cc1da34cf8ad30ae790cba/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L192): Merge this if statement with the enclosing one. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1066)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L95](https://github.com/ow2-proactive/scheduling-portal/blob/962554aa96e9eee886cc1da34cf8ad30ae790cba/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L95): Remove this useless assignment to local variable "msg". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L97](https://github.com/ow2-proactive/scheduling-portal/blob/962554aa96e9eee886cc1da34cf8ad30ae790cba/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L97): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L145](https://github.com/ow2-proactive/scheduling-portal/blob/962554aa96e9eee886cc1da34cf8ad30ae790cba/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L145): Remove this useless assignment to local variable "msg". [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
1. ![MAJOR][MAJOR] [ServerLogsController.java#L147](https://github.com/ow2-proactive/scheduling-portal/blob/962554aa96e9eee886cc1da34cf8ad30ae790cba/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/ServerLogsController.java#L147): Catch Exception instead of Throwable. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1181)
1. ![MAJOR][MAJOR] [AbstractOutputDisplayView.java#L88](https://github.com/ow2-proactive/scheduling-portal/blob/962554aa96e9eee886cc1da34cf8ad30ae790cba/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/AbstractOutputDisplayView.java#L88): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [OutputView.java#L93](https://github.com/ow2-proactive/scheduling-portal/blob/962554aa96e9eee886cc1da34cf8ad30ae790cba/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L93): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MAJOR][MAJOR] [OutputView.java#L105](https://github.com/ow2-proactive/scheduling-portal/blob/962554aa96e9eee886cc1da34cf8ad30ae790cba/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/view/OutputView.java#L105): Make this anonymous inner class a lambda [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1604)
1. ![MINOR][MINOR] [OutputController.java#L26](https://github.com/ow2-proactive/scheduling-portal/blob/962554aa96e9eee886cc1da34cf8ad30ae790cba/scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/controller/OutputController.java#L26): Rename this package name to match the regular expression '^[a-z]+(\.[a-z][a-z0-9]*)*$'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00120)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Pull request merged**